audiorecorder/src/test/java/org/odk/collect/audiorecorder/testsupport/RobolectricApplication.kt

```diff
@@ -0,0 +1,33 @@
+package org.odk.collect.audiorecorder.testsupport
+
+import android.app.Application
+import org.odk.collect.audiorecorder.AudioRecorderDependencyComponent
+import org.odk.collect.audiorecorder.AudioRecorderDependencyComponentProvider
+import org.odk.collect.audiorecorder.AudioRecorderDependencyModule
+import org.odk.collect.audiorecorder.DaggerAudioRecorderDependencyComponent
+
+/**
+ * Used as the Application in tests in in the `test/src` root. This is setup in `robolectric.properties`
+ */
+internal class RobolectricApplication : Application(), AudioRecorderDependencyComponentProvider {
```

Comment:
I'm a big fan of how this setup works in test. We end up with (in my mind) a much more intuitive setup where the `RobolectricApplication` has to do exactly the same thing as `Collect` does.